qadence_libs/qinfo_tools/spsa.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+import numpy as np
+import torch
+from qadence import Overlap
+from torch import Tensor
+
+
+def _create_random_direction(size: int) -> Tensor:
+    """
+    Creates a torch Tensor with elements randomly drawn from {-1,+1}.
+
+    Args:
+        size (int): Size of the vector
+    """
+    return torch.Tensor(np.random.choice([-1, 1], size=(size, 1)))
```

Comment:
```suggestion
    return torch.tensor(np.random.choice([-1, 1], size=(size, 1)))
```

Reply:
Actually, can't you use [torch.rand](https://pytorch.org/docs/stable/generated/torch.rand.html) ?